frame/composable-traits/src/vault.rs

```diff
@@ -55,8 +55,10 @@ where
 	AccountId: core::cmp::Ord,
 {
 	pub asset_id: CurrencyId,
+	/// Idle amount of assets for withdraw
```

Comment:
enum Parts(reserverd, stragegies) kind of. valuate sum to 1

Reply:
Sorry. Could you elaborate this statement?

Reply:
My apologies but I couldn't understand probably because I am still learning the codebase. Feel free to open an issue describing your concerns and I will try to tackle everything afterwards.

Reply:
everything should in sum be 100%. that is type. (idle, (strategy, amount))